crates/api_models/src/admin.rs

```diff
@@ -2160,6 +2176,14 @@ pub struct BusinessProfileResponse {
     /// These key-value pairs are sent as additional custom headers in the outgoing webhook request.
     #[schema(value_type = Option<Object>, example = r#"{ "key1": "value-1", "key2": "value-2" }"#)]
     pub outgoing_webhook_custom_http_headers: Option<HashMap<String, Secret<String>>>,
+
+    /// Merchant Connector id to be stored for tax_calculator connector
+    pub tax_connector_id: Option<String>,
+
+    /// Indicates if tax_calculator connector is enabled or not.
+    /// If set to `true` tax_connector_id will be checked.
+    #[serde(default)]
+    pub is_tax_connector_enabled: bool,
```

Comment:
We need not add `#[serde(default)]` for response types, if it's always expected to be returned by the application, and since the `BusinessProfile` domain model also has this field as a `bool` and not `Option<bool>`.